eclair-core/src/main/scala/fr/acinq/eclair/channel/Monitoring.scala

```diff
@@ -29,13 +30,19 @@ object Monitoring {
     val Event = "event"
     val Fatal = "fatal"
     val Origin = "origin"
+    val Status = "status"
 
     object Events {
       val Created = "created"
       val Closing = "closing"
       val Closed = "closed"
     }
 
+    object Statuses {
```

Comment:
Isn't that duplicated with `Events`? I think `States` is the proper name.

```suggestion
    object States {
```

Reply:
> Isn't that duplicated with Events?

It's somewhat but not completely duplicated, as `events` don't exactly map to channel states (since `created` isn't really a channel state). I think we'll need to keep both